corehq/apps/reports/standard/deployments.py

```diff
@@ -316,6 +331,7 @@ def process_rows(self, users, fmt_for_export=False):
             grouped_ancestor_locs = self.get_bulk_ancestors(location_ids)
             self.required_loc_columns = self.get_location_columns(grouped_ancestor_locs)
 
+        user_loc_dict = self.user_locations_dict(users)
```

Comment:
nit: and then changing this variables name as well.

Reply:
[ba975bf](https://github.com/dimagi/commcare-hq/pull/34744/commits/ba975bf2ee0e9031d8d1293347297ef32ba8bf03)